### PR TITLE
feature/bulk-publish-v2

### PR DESCRIPTION
**Pull request by markkaylor**
This is the feature branch for work on bulk-publish-v2. Reviewed and QA'd PR's are merged into this branch. This PR does not need a review.

### CODE REVIEW

**Comment by joshuaellis** [comment]
wondering if we should squash this before it's merged, i'm concerned the git history is messy and will be _painful_ should we need to untangle in the future – what do you think folks?

**Comment by markkaylor** [comment]
@joshuaellis yes we can squash before merging this and start making it a point to squash before merging into it :)

File: packages/core/admin/admin/src/constants.js [hunk 1]
```diff
@@ -1,90 +1,90 @@
 export const ADMIN_PERMISSIONS_CE = {
```
Comment (markkaylor) [inline packages/core/admin/admin/src/constants.js:1]: Not sure why we have this diff here but pretty sure it is the same as main

File: packages/core/admin/admin/src/pages/ProfilePage/index.js [hunk 2]
```diff
@@ -203,7 +203,12 @@ const ProfilePage = () => {
               <HeaderLayout
```
Comment (markkaylor) [inline packages/core/admin/admin/src/pages/ProfilePage/index.js:203]: Not sure why we have this diff but it looks just like a formatting change

File: packages/core/admin/ee/admin/constants.js [hunk 3]
```diff
@@ -1,16 +1,16 @@
 export const ADMIN_PERMISSIONS_EE = {
-    settings: {
```
Comment (markkaylor) [inline packages/core/admin/ee/admin/constants.js:2]: Not sure why we have this diff, will double check

Comment (markkaylor) [reply packages/core/admin/ee/admin/constants.js:2]: Also just a formatting diff for indentation the content is the same

**Review by simotae14 — APPROVED** [review]
LGTM

**Pull request merged**